### DISCUSSION TITLE
Strange difference between Bright app consumption and that reported in HA

### DISCUSSION THREAD

**Discussion by ronnyTodgers** [HandyHat/ha-hildebrandglow-dcc#300] [General]
Have only just got this setup today, thank you for an awesome plugin!

I'm noticing that the usage I see in the bright app ( and in fact loop ) is quite different to the usage that is reported in HA- more accurately the totals are correct but the timing and shape of the curve is different - I've checked the time date settings in docker and HASS - both correct ( I'm in GMT ) the account is definitely correct and the totals match so that would be a stretch.

Any pointers for where I might find a solution to this?

I'm on version 1.0.3 of the plugin and 2023.1.5 of HA running in Docker

![Screenshot_20230118-172521](https://user-images.githubusercontent.com/22333275/213252978-0257b98b-fa70-4d8c-9e2c-7ec4b0191f98.png)

![image](https://user-images.githubusercontent.com/22333275/213253098-eeb4a1a0-8b2c-4de8-a8f4-3e25706ecc91.png)

**Comment by trevorwarwick** [comment]
I see the same thing, and I'm guessing it's to do with how the Hildebrand API works. If HA polls for data and doesn't get any because the Hildebrand API service is overloaded, then the amounts effectively just get rolled up until it does successfully connect.
